### PR TITLE
Added initial unit test to check file upload

### PR DESCRIPTION
**Pull request by utsavgarg**
Added a test to check if file upload is working correctly

### CODE REVIEW

File: caffe_app/tests.py [hunk 1]
```diff
@@ -1 +1,16 @@
-# Create your tests here.
+from django.test import Client
```
Comment (deshraj) [inline caffe_app/tests.py]: Can we have imports grouped in alphabetical order? For reference, see this https://stackoverflow.com/questions/20762662/whats-the-correct-way-to-sort-python-import-x-and-from-x-import-y-statement?answertab=active#tab-top

File: caffe_app/tests.py [hunk 2]
```diff
@@ -1 +1,16 @@
...
+import os
+
+
+class UploadTest(unittest.TestCase):
+    def setUp(self):
+        self.client = Client()
+
+    def test_details(self):
+        sample_file = open(os.path.join(settings.BASE_DIR, 'example', 'GoogleNet.prototxt'), 'r')
+        response = self.client.post('/caffe/import', {'file': sample_file})
```
Comment (deshraj) [inline caffe_app/tests.py]: Instead of using the hard coded URL `/caffe/import`, please use `reverse()`

File: caffe_app/tests.py [hunk 3]
```diff
@@ -1 +1,16 @@
...
+        response = json.loads(response.content)
+        print 'Test {:}!'.format(response['result'])
```
Comment (deshraj) [inline caffe_app/tests.py]: Please remove debug statement.

Comment (deshraj) [inline caffe_app/tests.py]: Also, I cannot find any assert statements.

Comment (utsavgarg) [inline caffe_app/tests.py]: Assert statement to check the status to be success ?

File: tests/unit/caffe_app/test_views.py [hunk 4]
```diff
@@ -0,0 +1,934 @@
...
+                                  transform_param=dict(crop_size=227, mean_value=[104, 117, 123],
+                                                       mirror=True, force_color=False,
+                                                       force_gray=False))
+        with open(os.path.join(settings.BASE_DIR, 'media', 'test.prototxt'), 'w') as f:
+            f.write(str(to_proto(data, label)))
+        sample_file = open(os.path.join(settings.BASE_DIR, 'media', 'test.prototxt'), 'r')
+        response = self.client.post(reverse('caffe-import'), {'file': sample_file})
+        response = json.loads(response.content)
+        os.remove(os.path.join(settings.BASE_DIR, 'media', 'test.prototxt'))
+        self.assertTrue(len(response['net']['l0']['params']) >= 13)
```
Comment (deshraj) [inline tests/unit/caffe_app/test_views.py]: Please use https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertGreaterEqual

File: tests/unit/caffe_app/test_views.py [hunk 5]
```diff
@@ -0,0 +1,934 @@
...
+from django.conf import settings
+from django.core.urlresolvers import reverse
+from django.test import Client
+
+
+class ImportPrototxtTest(unittest.TestCase):
+    def setUp(self):
+        self.client = Client()
+
+    def test_details(self):
```
Comment (deshraj) [inline tests/unit/caffe_app/test_views.py]: replace `test_details` with `test_<view name comes here>`

File: tests/unit/tensorflow_app/test_views.py [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+from django.conf import settings
+from django.core.urlresolvers import reverse
+from django.test import Client
+
+
+class UploadTest(unittest.TestCase):
+    def setUp(self):
+        self.client = Client()
+
+    def test_details(self):
```
Comment (deshraj) [inline tests/unit/tensorflow_app/test_views.py]: Please replace `test_details` with `test_<view name comes here>`

File: tests/unit/caffe_app/test_views.py (hunk 5 above)
Comment (utsavgarg) [inline tests/unit/caffe_app/test_views.py]: so `test_details` becomes `test_caffe-import` ?

Comment (utsavgarg) [inline tests/unit/caffe_app/test_views.py]: so `test_details` becomes `test_caffe-import` ?

**Comment by deshraj** [comment]
Awesome! good job @utsavgarg :)

**Pull request merged**